3-ML-Basic.sh

```diff
@@ -58,22 +57,26 @@ if [[ $(command -v conda) || (-n $CIINSTALL) ]]; then
     PIP="pip install"
 else
     execute sudo apt-get update
-    # execute sudo apt-get install python3 python3-dev python-dev python-tk -y
-    # if [[ ! -n $CIINSTALL ]]; then execute sudo apt-get install python3-pip python-pip; fi
     PIP="sudo pip3 install"
 fi
 
 execute sudo apt-get install libhdf5-dev
 
 # Install opencv from pip only if it isn't already installed. Need to use `pkgutil` because opencv built from source does not appear in `pip list`
-if [[ $(python -c "import pkgutil; print([p[1] for p in pkgutil.iter_modules()])" | grep cv2) ]]; then
+if [[ ! $(python -c "import pkgutil; print([p[1] for p in pkgutil.iter_modules()])" | grep cv2) ]]; then
```

Comment:
`python` will work only if Anaconda is being used, otherwise `python3` should be used